src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
Link to CSS should have an extension of `.css` not `.scss`. SCSS is a preprocessor and needs to be compiled into CSS for the browser to understand. Ensure you link to the compiled CSS file.